### PR TITLE
fix(IDLIX): fix presence not working

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

- Fix issues that were created by site changes & by #8044
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot_18

</details>

### CODE REVIEW

File: websites/I/IDLIX/metadata.json [hunk 1]
```diff
@@ -8,20 +8,16 @@
...
-	"tags": [
-		"video",
-		"anime",
-		"indonesian",
-		"idlix"
-	],
+	"tags": ["video", "anime", "indonesian", "idlix"],
 	"iframe": true,
-	"iFrameRegExp": "https://jeniusplay[.]com",
+	"iFrameRegExp": ".*",
```
Comment (DarkVillager) [inline websites/I/IDLIX/metadata.json:20]: They use 10+ different services now.

File: websites/I/IDLIX/metadata.json [hunk 2]
```diff
@@ -8,20 +8,16 @@
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "tv.idlixofficial.co",
-	"version": "1.0.9",
+	"url": "idlix.online",
+	"regExp": "^([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31))(?<!127)(?<!^10)(?<!^0).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!192.168)(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})",
```
Comment (DarkVillager) [inline websites/I/IDLIX/metadata.json:12]: Use [htis regex](https://stackoverflow.com/questions/33453057/regex-to-only-match-public-ipv4-address) to match all public ips (bcuz they use random public ips.)

File: websites/I/IDLIX/metadata.json [hunk 3]
```diff
@@ -8,20 +8,16 @@
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "tv.idlixofficial.co",
-	"version": "1.0.9",
+	"url": "tv1.idlixofficial.co",
+	"regExp": "^([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31))(?<!127)(?<!^10)(?<!^0).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(?<!192.168)(?<!172.(16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31)).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]).([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})",
```
Comment (DarkVillager) [inline websites/I/IDLIX/metadata.json:12]:
```suggestion
	"regExp": "/(\d+)(?<!10)\.(\d+)(?<!192\.168)(?<!172\.(1[6-9]|2\d|3[0-1]))\.(\d+)\.(\d+)|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})/gm",
```

File: websites/I/IDLIX/metadata.json [hunk 4]
```diff
@@ -8,20 +8,16 @@
 	"description": {
 		"en": "Watch the latest movies, TV series, dramas, anime with high quality available in Indonesian subtitles and updated every day. Box Office Movies to the Best TV Series are all available on the IDLIX site."
 	},
-	"url": "tv.idlixofficial.co",
-	"version": "1.0.9",
+	"url": "tv1.idlixofficial.co",
+	"regExp": "/(\d+)(?<!10)\.(\d+)(?<!192\.168)(?<!172\.(1[6-9]|2\d|3[0-1]))\.(\d+)\.(\d+)|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})/gm",
```
Comment (DarkVillager) [inline websites/I/IDLIX/metadata.json:12]:
```suggestion
	"regExp": "(\d+)(?<!10)\.(\d+)(?<!192\.168)(?<!172\.(1[6-9]|2\d|3[0-1]))\.(\d+)\.(\d+)|((tv([0-9])[.])?id(f)?lix(official)?[.][a-z]{2,6})",
```

**Review by Bas950 — APPROVED** [review]

**Pull request merged**